src/components/OutputPreview.tsx

```diff
@@ -20,6 +22,7 @@ interface Props {
 
 export const OutputPreview = ({ columnQueries, filteredJson }: Props) => {
 	const [format, setFormat] = useState<OutputFormat>(OutputFormat.CSV);
+	const [value, copy] = useCopyToClipboard();
```

Comment:
```suggestion
	const [, copy] = useCopyToClipboard();
```

Since this value isn't being used, it's safe to ignore in the destructuring